packages/background/src/utils/ethereumChain.ts

```diff
@@ -44,9 +44,6 @@ export const formatAndValidateRpcURL = (rpcUrl: string): string => {
     if (!rpcUrl) {
         throw new Error('No RPC endpoint provided');
     }
-    if (rpcUrl.indexOf('https://') === -1) {
```

Comment:
This is a blocker if the user wants to add custom nodes using a local configurations (localhost, private IP using a node in a container, etc) because it will probably use http. In fact our default rpc url in the local node uses http instead of https:
![image](https://user-images.githubusercontent.com/98833290/199976553-5b67584f-c783-4565-9071-2275401e0e70.png)